SonarTfsAnnotate/Program.cs

```diff
@@ -73,16 +73,20 @@ static int Main(string[] args)
                         continue;
                     }
 
+                    WorkspaceInfo workspaceInfo = Workstation.Current.GetLocalWorkspaceInfo(path);
```

Comment:
@dbolkensteyn Saleha is still working on this issue. With just a file path we cannot get the workspace info if the mapping is not in sync. The two options i can think of are 1.) Get the tfs collection url from the user & ensure update workspace info before starting annotation. 2.) Ensure Update for all the LocalWorkspaces fetched from Workstation.Current.GetAllLocalWorkspaceInfo.